crates/fuzzing/src/generators/stacks.rs

```diff
@@ -50,7 +51,10 @@ impl Stacks {
         let mut work_list = vec![0];
 
         while let Some(f) = work_list.pop() {
-            let mut ops = u.arbitrary::<Vec<Op>>()?;
+            let mut ops = u
+                .arbitrary_iter::<Op>()?
+                .take(MAX_OPS)
+                .collect::<Result<Vec<_>>>()?;
             for op in &mut ops {
```

Comment:
My impression is it would be better to use `Unstructured::arbitrary_len`, to make the best use of the random input and give the fuzzer better opportunities to manipulate the generated objecs:

```suggestion
            let mut ops = Vec::new();
            for _ in 0..u.arbitrary_len()?.min(MAX_OPS) {
                let mut op = u.arbitrary()?;
```

(And an `ops.push(op);` at the end.)

Reply:
Sure thing